client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -203,6 +204,10 @@ export default React.createClass( {
 			.forEach( site => action( site, site.plugin ) );
 	},
 
+	pluginHasUpdate( plugin )  {
```

Comment:
I removed it from `plugin-item` because I want this component to have access to that logic as well. It needs to be able to re-order the array of plugins based on whether a plugin requires an update. Thanks for taking a look :)